### ISSUE TITLE
Can't put "long" key

### ISSUE THREAD

**Issue by Stajor** [aerospike/aerospike-client-php#35]
Can't put new row with key longer than 14 chars 
```
$data = ['my1234567890123' => 'value'];
$connection->put('my_set', $data);
```
error from aerospike: Unable to set record to int events 
Also I can't store key started from integer

**Comment by rbotzer** [comment]
There's a built-in limit of 14 characters on bin names (See the [FAQ](https://www.aerospike.com/docs/guide/FAQ.html)). I was sure I already added this to the documentation...apparently I check-marked it off in my head before really doing so. Sorry about that.

**Comment by rbotzer** [comment]
The upcoming release of PHP will have a clear error message for such a case.

**Comment by rbotzer** [comment]
Fixed by release 3.3.12. :rocket:
